examples/tornjak/README.md

```diff
@@ -42,35 +52,70 @@ You can now access Tornjak at [localhost:3000](http://localhost:3000).
 
 See [values.yaml](./values.yaml) for more details on the chart configurations to achieve this setup.
 
-## Tornjak and Ingress with ingress-nginx
+## Deploy Tornjak with ingress-nginx
 
 Update examples/production/example-your-values.yaml with your information, most importantly, trustDomain.
 
 ```shell
-helm upgrade --install --namespace spire-server spire charts/spire \
---values examples/production/values.yaml \
+helm upgrade --install -n spire-mgmt spire spire \
+--repo https://spiffe.github.io/helm-charts-hardened/ \
 --values examples/tornjak/values.yaml \
 --values examples/tornjak/values-ingress.yaml \
 --set global.spire.ingressControllerType=ingress-nginx \
---values examples/production/example-your-values.yaml \
---render-subchart-notes --debug
+--values your-values.yaml \
+--render-subchart-notes
 ```
 
 ## Tornjak and Ingress on Openshift
 
-When deploying on Openshift, follow the deployment setup as described in
-[Openshift README](../openshift/README.md)
+Obtain the OpenShift Apps Subdomain for Ingress
+We will be using the OpenShift application subdomain during our deployment, so let's capture it now and create environment variable by executing the following command:
 
-Then just add Openshift specific configuration to the above command:
+```shell
+export appdomain=$(oc get cm -n openshift-config-managed  console-public -o go-template="{{ .data.consoleURL }}" | sed 's@https://@@; s/^[^.]*\.//')
+echo $appdomain
+```
```

Comment:
Should we repeat this here since the main install instructions cover openshift now?

Same question with all the global.openshift / global.spire stuff below.

Reply:
I didn't see the reference to Openshift under the current documentation, but regardless, I think it is a good idea to show it here since the users would be doing an helm upgrade and they need the Ingress info to continue.